src/Controller/Admin/UserController.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\User;
+use App\Form\Type\Admin\CreateNewUserType;
+use App\Form\Type\Admin\UpdateUserType;
+use App\Repository\UserRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class UserController extends AbstractController
+{
+    private UserRepository $userRepository;
+
+    private EntityManagerInterface $entityManager;
+
+    private UserPasswordHasherInterface $userPasswordHasher;
+
+    private int $defaultPerPage;
+
+    public function __construct(
+        UserRepository $userRepository,
+        EntityManagerInterface $entityManager,
+        UserPasswordHasherInterface $userPasswordHasher,
+        string $defaultPerPage
+    ) {
+        $this->userRepository = $userRepository;
+        $this->entityManager = $entityManager;
+        $this->userPasswordHasher = $userPasswordHasher;
+        $this->defaultPerPage = (int) $defaultPerPage;
+    }
+
+    /**
+     * @Route("admin/users", name="admin_users", methods={"GET"})
+     */
+    public function getAllUsers(Request $request): Response
+    {
+        $paginate['currentPage'] = $request->query->get('page', 1);
+        $paginate['perPage'] = $request->query->get('perPage', $this->defaultPerPage);
+
+        $result = $this->userRepository->showUsersPaginated($paginate);
+
+        $users = $this->userRepository->findAll();
+
+        return $this->render('Admin/allUsers.html.twig', [
+            'users' => $users
+//            [max(10)]
+//            'paginate' => $paginate,
+//            'result' => $result
+        ]);
+    }
+
+    /**
+     * @Route("admin/user/{id}", name="user_update", requirements={"id"="\d+"})
+     */
+    public function updateUser(Request $request, int $id): Response
+    {
+        $user = $this->userRepository->findOneBy(['id' => $id]);
+        if (null === $user) {
+            $this->addFlash(
+                'error',
+                'User does not exist!'
+            );
+
+            return new Response('User does not exist', Response::HTTP_NOT_FOUND);
```

Comment:
you are in the admin area, you should redirect to user listing, not return a blank page with `User does not exist`